inference-engine/CMakeLists.txt

```diff
@@ -79,7 +79,7 @@ function(ie_build_samples)
                 MINGW64 CMAKE_BUILD_TYPE CMAKE_MACOSX_RPATH)
         unset(${var})
     endforeach()
-
+    include(sanitizer)
```

Comment:
why do we need it?

Reply:
samples must link with sanitizer flags `-fsanitize-thread -fuse=ldd` for sanitizer builds so that linker configures dependencies correctly. Line 74 and 75 above clean build flags and here we set back required.